cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -192,6 +192,70 @@ bool ImageFlip::convertImageMessageToMat(const sensor_msgs::Image::ConstPtr& ima
 }
 
 
+template <typename T> void ImageFlip::atForDiffImTypes(cv::Mat& image, int row_index, int column_index, double value, bool replace_prev_value)
```

Comment:
Correct indentation in this file as well, i.e. tabs instead of spaces and correct number of tabs.